packages/cli/README.md

```diff
@@ -93,7 +99,7 @@ _If you're looking for programmatic usage, then refer to the underlying [@code-p
 
 6. View report file(s) in output directory (specified by `persist.outputDir` configuration).
 
-## Portal integration
+## 🌐 Portal integration
```

Comment:
Adding the emojis breaks your links :warning: Notice that GitHub uses `#-portal-integration` instead `#portal-integration` now: https://github.com/code-pushup/cli/blob/docs-cli-improvements/packages/cli/README.md#-portal-integration

Reply:
done